.github/workflows/build.yml

```diff
@@ -77,18 +64,16 @@ jobs:
                     fi
                     
                     echo "Tag $DOCKER_TAG"
```

Comment:
`$DOCKER_TAG` is currently not right, as it doesn't consider the target (supervisor, debug).